clothing_rental/clothing_rental/settings.py

```diff
@@ -19,6 +19,7 @@
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
+TEMPLATE_DIR = os.path.join(BASE_DIR,"templates")
```

Comment:
Jak już korzystamy z pathliba i Path, to możemy dać `BASE_DIR / "your_directory"`

Reply:
to znaczy TEMPLATE_DIR = BASE_DIR / "templates" ?

Reply:
tak